src/Compilers/CSharp/Portable/Symbols/Source/SourceEventSymbol.cs

```diff
@@ -180,7 +203,7 @@ private CustomAttributesBag<CSharpAttributeData> GetAttributesBag()
                 LoadAndValidateAttributes(OneOrMany.Create(this.AttributeDeclarationSyntaxList), ref _lazyCustomAttributesBag))
```

Comment:
I think the key problem I don't like here is that we use `_lazyCustomAttributesBag` to indicate that we're done loading attributes without checking `CompletionPart.Attributes` as well.